charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
-appVersion: v2.51.1
-version: 25.19.1
+appVersion: v2.51.2
+version: 25.19.2
```

I'd suggest that we indicate a minor change due to changes in the pushgateway subchart.

```suggestion
version: 25.20.0
```